packages/cli/package.json

```diff
@@ -41,26 +41,27 @@
     "bn.js": "^5.0.0",
     "chalk": "^2.4.2",
     "cli-table": "^0.3.1",
-    "cli-ux": "^5.3.1",
+    "cli-ux": "5.3.1",
```

Comment:
Why have you locked the version to 5.3.1?

Reply:
I used snyk tool to fix the versions. I guess it locked the version. I've changed that.